cmd/nri-prometheus/config.go

```diff
@@ -4,25 +4,46 @@ package main
 
 import (
 	"fmt"
+	"path/filepath"
 	"reflect"
 	"regexp"
 	"strings"
 	"time"
 
+	"github.com/newrelic/infra-integrations-sdk/args"
 	"github.com/newrelic/nri-prometheus/internal/cmd/scraper"
 	"github.com/pkg/errors"
 	"github.com/spf13/viper"
 )
 
+// ArgumentList Available Arguments
+type ArgumentList struct {
+	ConfigPath string `default:"" help:"Path to the config file"`
+}
+
 func loadConfig() (*scraper.Config, error) {
+
+	c := ArgumentList{}
+	err := args.SetupArgs(&c)
```

Comment:
Please test that using the infra integrations sdk args it's not interfering with Viper.

Reply:
I checked the code and ArgumentList does not interfere with Viper.
I also tested manually locally and in the container I got from the Dockerfile included